<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_send_kpid">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Kpid</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_gain_kp">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>42</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_gain_ki">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_gain_kd">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_kp">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ki">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ki</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_kd">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>300</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kd</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_req_frequancy">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>10000</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_fed_frequancy">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_req_frequancy">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>22</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_req_frequancy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>REQUIRED FREQUANCY:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fed_frequancy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FEEDBACK FREQUANCY:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_pid_upid">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_fed_frequancy_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>PID Upid VALUE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTst">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kp</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTsta">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>430</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
